<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerGetDialog</class>
 <widget class="QDialog" name="ServerGetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #DFE8CC ;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="teknikServis">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Teknik Servis</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QGroupBox" name="DonanimBox">
               <property name="minimumSize">
                <size>
                 <width>400</width>
                 <height>175</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>18</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Cihazla gelen donanımlar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_17">
                  <item>
                   <widget class="QCheckBox" name="donanim_1">
                    <property name="font">
                     <font>
                      <pointsize>18</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Power Kablo</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="donanim_1Num">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>16</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_16">
                  <item>
                   <widget class="QCheckBox" name="donanim_2">
                    <property name="font">
                     <font>
                      <pointsize>18</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Saçak Kablo</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="donanim_2Num">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>16</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_18">
                  <item>
                   <widget class="QCheckBox" name="donanim_3">
                    <property name="font">
                     <font>
                      <pointsize>18</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>BNC Kablo</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="donanim_3Num">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>16</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="olayBox">
               <property name="minimumSize">
                <size>
                 <width>250</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_30">
                <item>
                 <widget class="QLabel" name="olay">
                  <property name="font">
                   <font>
                    <pointsize>18</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Olay</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="olay_">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>20</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QGroupBox" name="TestBox">
               <property name="styleSheet">
                <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <widget class="QLabel" name="testSuresi">
                  <property name="font">
                   <font>
                    <pointsize>18</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Test Süresi(saat)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="testSuresi_">
                  <property name="font">
                   <font>
                    <pointsize>18</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="styleSheet">
                <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_13">
                <item>
                 <widget class="QLabel" name="gelisTarihi">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>18</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Tarih</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="spacing">
                   <number>2</number>
                  </property>
                  <item>
                   <widget class="QSpinBox" name="gelisTarihiDay_">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>18</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="gelisTarihiMonth_">
                    <property name="font">
                     <font>
                      <pointsize>18</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="gelisTarihiYear_">
                    <property name="font">
                     <font>
                      <pointsize>18</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_14">
                  <item>
                   <widget class="QLabel" name="label_6">
                    <property name="font">
                     <font>
                      <pointsize>18</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Saat</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <property name="spacing">
                     <number>2</number>
                    </property>
                    <item>
                     <widget class="QSpinBox" name="tarih_saat">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>18</pointsize>
                       </font>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="tarih_dakika">
                      <property name="font">
                       <font>
                        <pointsize>18</pointsize>
                       </font>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QGroupBox" name="YapilanIslemBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>270</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QGroupBox {
border: 2px solid grey;
}</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QLabel" name="yapilanIslem">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Yapılan İşlem / Arıza Tarifi</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="yapilanIslem_">
                <property name="minimumSize">
                 <size>
                  <width>600</width>
                  <height>120</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="verticalGroupBox">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Not Ekle</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="notlar_">
              <property name="minimumSize">
               <size>
                <width>450</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>510</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <pointsize>28</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Güncelle</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
